dao/src/main/java/com/epam/esm/config/DaoConfig.java

```diff
@@ -0,0 +1,55 @@
+package com.epam.esm.config;
+
+import com.epam.esm.entity.GiftCertificate;
+import com.epam.esm.entity.Tag;
+import com.epam.esm.mappers.GiftCertificateRowMapper;
+import com.epam.esm.mappers.TagRowMapper;
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+import org.springframework.context.annotation.*;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.datasource.DataSourceTransactionManager;
+import org.springframework.transaction.PlatformTransactionManager;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+import javax.sql.DataSource;
+
+@Profile("dev")
+@Configuration
+@EnableTransactionManagement
+@PropertySource("classpath:hikari.properties")
+@ComponentScan(basePackages = {"com.epam.esm.dao"})
+public class DaoConfig {
+
+    private final static String HIKARI_PROPERTIES_PATH = "/hikari.properties";
+
+    @Bean(name = "transactionManager")
+    public PlatformTransactionManager getTransactionManager() {
+        return new DataSourceTransactionManager(getDataSource());
+    }
+
+    @Bean
+    public RowMapper<GiftCertificate> getGiftCertificateRowMapper() {
```

Comment:
@gkttk Unnecessary bean definition. You can add `@Component` for your mapper.

Reply:
commit 394a39d